attributes/default.rb

```diff
@@ -22,6 +22,10 @@
 default['librenms']['phpini']['memory_limit'] = '128M'
 default['librenms']['phpini']['timezone'] = 'UTC'
 
+default['librenms']['web']['enablessl'] = false
```

Comment:
How to you plan to create those certificates? Because currently those are not even present on a freshly installed server ?

Reply:
I am currently using a forked copy of ssl-vault cookbook, which unfortunately hasn't been updated in years, to install a cert/key pair I created and placed in vault. I wasn't sure how everyone else was handling certs so I thinking that it would be up to their wrapper cookbook, etc, to create and/or place the cert/key.

Reply:
I have been thinking about this and maybe a better way to do this would be to have an attribute that could be set to a new template file instead of using the default web_app.conf.erb. Then you would setup SSL how every you wanted or not at all and we wouldn't need any attributes for anything SSL related. It would also work for others that wanted to do anything else that couldn't be controlled in only the directory section of the template.